src/test/java/seedu/address/logic/parser/AbcliParserTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ExitCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.SwitchParserModeCommand;
+import seedu.address.logic.commands.meetup.AddCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.buyer.Buyer;
+import seedu.address.model.meetup.MeetUp;
+import seedu.address.testutil.buyer.BuyerBuilder;
+import seedu.address.testutil.buyer.BuyerUtil;
+import seedu.address.testutil.meetup.MeetUpBuilder;
+import seedu.address.testutil.meetup.MeetUpUtil;
+
+public class AbcliParserTest {
```

Comment:
Good code for testing, test cases are vigorous and considers the many different cases. LGTM!